examples/hacker_news_pwa/web/index.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <meta name="theme-color" content="#212121">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="Description" content="An implementation of a Hacker News readers as a Progressive Web Apps using the AngularDart framework.">
```

Comment:
Suggestion:
"A Hacker News reader implemented as a Progressive Web App using AngularDart."

Otherwise, if you'd prefer to keep the original description, "readers" and "Apps" shouldn't be plural.